apps/app/src/views/routes/Settings/components/users/RemoveUserActionItem.tsx

```diff
@@ -0,0 +1,99 @@
+// RemoveUserMenuItem.tsx
+import React from 'react';
+import { MenuItem, useColorMode, useToast } from '@chakra-ui/react';
+import { FragmentType, graphql, useFragment } from 'apps/app/src/gql';
+import { FiTrash } from 'react-icons/fi';
+import { confirmWrapper } from '../../../../components/GuardDialog';
+import { useMutation } from '@apollo/client';
+import { usePhoton } from '@photonhealth/react';
+import { StyledToast } from 'apps/app/src/views/components/StyledToast';
+
+export const userFragment = graphql(/* GraphQL */ `
+  fragment UserFragment on User {
+    id
+    npi
+    phone
+    fax
+    email
+    address {
+      street1
+      street2
+      state
+      postalCode
+      country
+      city
+    }
+    name {
+      first
+      full
+      last
+      middle
+      title
+    }
+    roles {
+      description
+      id
+      name
+    }
+  }
+`);
```

Comment:
I dont think you need all of these? Looks like you only need `id` and `name`. 

Also, the fragment name should be `RemoveUserActionItemUserFragment`. And then you need to actually use that fragment